client/my-sites/pages/page-list.jsx

```diff
@@ -42,17 +42,6 @@ export default class PageList extends Component {
 		page: 1,
 	};
 
-	// @TODO: Please update https://github.com/Automattic/wp-calypso/issues/58453 if you are refactoring away from UNSAFE_* lifecycle methods!
-	UNSAFE_componentWillReceiveProps( nextProps ) {
-		if (
-			nextProps.search !== this.props.search ||
-			nextProps.siteId !== this.props.siteId ||
-			nextProps.status !== this.props.status
-		) {
-			this.resetPage();
```

Comment:
This is the last usage of the `resetPage` method, so we can actually delete it now.